modules/errors.go

```diff
@@ -0,0 +1,12 @@
+package modules
```

Comment:
not sure creating a whole errors file is best, I think it's probably better to put these lines into the `renter.go` file since this error is really just relevant to the renter.